jetty-ee11/jetty-ee11-servlet/src/main/java/org/eclipse/jetty/ee11/servlet/ServletChannelState.java

```diff
@@ -1054,7 +1064,7 @@ public void sendError(int code, String message)
             request.setAttribute(ERROR_ORIGIN, servletContextRequest.getServletName());
             request.setAttribute(ErrorHandler.ERROR_CONTEXT, servletContextRequest.getServletContext());
             request.setAttribute(ErrorHandler.ERROR_MESSAGE, message);
-            request.setAttribute(ErrorHandler.ERROR_STATUS, code);
+            request.setAttribute(ERROR_STATUS, code);
```

Comment:
These attributes should all either be referenced via the `ErrorHandler` or none for consistency.